tests/test_frontend/test-i18n/test-i18n-src.json

```diff
@@ -1,6 +1,11 @@
 {
   "_global.hello_global": "Hello, global!",
   "_global.hello_global_repeat": "hello, global!",
+  "_global.incorrectly-formatted-key": "This key isn't formatted correctly",
```

Comment:
Add a few keys for future testing, @OmarAI2003 :) `_global.incorrectly-formatted-key` needs to be in snake case and `_global.incorrectly_named_key` is only used in `test-file.ts` so it should be `test_file.incorrectly_named_key`.

Reply:
Just FYI on the changes :)